extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/DevServicesKafkaProcessor.java

```diff
@@ -101,10 +107,12 @@ public void run() {
         cfg = configuration;
 
         if (bootstrapServers != null) {
-            log.infof(
-                    "Dev Services for Kafka started. Start applications that need to use the same Kafka broker "
-                            + "using -Dkafka.bootstrap.servers=%s",
-                    bootstrapServers.getBootstrapServers());
+            if (kafkaBroker.isOwner()) {
+                log.infof(
+                        "Dev Services for Kafka started. Other Quarkus applications in dev mode will find the broker automatically. To make sure to use the same Kafka broker, "
```

Comment:
I think this is a bit confusing, as you say it works automatically then say you need the property. I would say something like 'For Quarkus applications in production mode you can connect to this by starting your application with ...'

Reply:
Good point!